src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -48,7 +48,49 @@ public class SendReceiveTest : ClientWebSocketTestBase
                     recvResult = await cws.ReceiveAsync(receiveSegment, ctsDefault.Token);
                 }
 
-                await cws.CloseAsync(WebSocketCloseStatus.NormalClosure, "PartialMessageTest", ctsDefault.Token);
+                await cws.CloseAsync(WebSocketCloseStatus.NormalClosure, "PartialMessageDueToSmallReceiveBufferTest", ctsDefault.Token);
+            }
+        }
+
+        [ActiveIssue(21102)]
```

Comment:
Thanks for patching the cloud service, David! I'll update the ActiveIssue and re-run the ClientWebSocket tests against corefx-net.cloudapp.net with both frameworks (.NET Core and UWP).